src/framework/edge.ml

```diff
@@ -1,3 +1,6 @@
+(** CFG edge.
+    Corresponds to a (primitive) program statement between program points (and their states). *)
```

Comment:
how do they correspond to states?

Reply:
There's another fairly common CFG representation where statements are in nodes (not edges) and states are moving between them via edges (not held in nodes).

We need to make this very clear because our nodes are also identified by a CIL statement ID, but we don't use such a CFG representation.